src/pkg/utils/utils.go

```diff
@@ -497,10 +497,15 @@ func PathExists(path string) bool {
 }
 
 // IsContainerNameValid checks if the name of a container matches the right pattern
-func IsContainerNameValid(containerName string) (bool, error) {
+func IsContainerNameValid(containerName string) bool {
 	pattern := "^" + ContainerNameRegexp + "$"
 	matched, err := regexp.MatchString(pattern, containerName)
-	return matched, err
+	if err != nil {
+		logrus.Warnf("There was an error while matching container name: %v", err)
```

Comment:
In this case, I think we should just `panic` because this can only happen due to a programming mistake, and not some likely run-time condition that should be handled or logged. It's similar to an assertion failure -- something that should never happen if the code is sane.